### ISSUE TITLE
install qgis will unintall vts-backend

### ISSUE THREAD

**Issue by veyvin** [Melown/vts-backend#7]

**Comment by vaclavblazek** [comment]
Hi, could you please provide more info? What system are you working with? Could you paste here package manager output (what it installs, upgrades, removes).

**Comment by veyvin** [comment]
@vaclavblazek  when i add https://launchpad.net/~ubuntugis/+archive/ubuntugis-unstable     PPA,  and install qgis , the vts-backend remove

**Comment by vaclavblazek** [comment]
TL;DR you can always install **qgis** from **upstream**: https://qgis.org/en/site/forusers/alldownloads.html#debian-ubuntu . This package is built against vanilla Ubuntu and we've successfully installed it alongside the vts-backend suite.

Regarding ubuntugis unstable:

No, you cannot do that. Unstable ubuntugis is not supported to co-exist with our package repository.

The thing is that GDAL C++ library (a qgis dependency) is not forward compatible and upstream doesn't increase library SO version. When I pointed that problem to them they told me that they are more concerned on C API version not needing upgrade than to allow multiple GDAL versions alongside.

Just installing `libgdal20` from ubuntugis unstable (i.e. upgrading it from version 2.2.3 to version 2.4.0) will break `vts-vtsd` and `vts-mapproxy`, if it were allowed by package manager.

If you really need to use ubuntugis unstable and feel brave you can try to build `vts-mapproxy` and `vts-vtsd` and `vts-backend` from sources.
